app/services/atmosphere/cloud/scale_appliance.rb

```diff
@@ -0,0 +1,51 @@
+module Atmosphere
```

Comment:
Done in https://github.com/dice-cyfronet/atmosphere/commit/8748bc671e41fb67a1674c8e01ac53f31bc68de4